query_tests/generate_rnd_queries.py

```diff
@@ -74,23 +89,36 @@
 IMPORT_DATA = "COPY benchmarks.query_tests FROM 's3://crate-stresstest-data/query-tests/*.json'"
 
 
+def expr_with_operator(column, provider, inner_type, dims):
+    val = provider()
+    if inner_type in ('string', 'ip'):
+        val = f"'{val}'"
+    operators = OPERATORS_BY_TYPE.get(inner_type)
+    if not operators or random.randint(1, 20) == 1:
```

Comment:
`random.randint(1, x) == 1` could be refactored into a separate method, because it will be used more often in the future.